pkg/steps/censoringclient/client.go

```diff
@@ -0,0 +1,138 @@
+package censoringclient
```

Comment:
The name is IMHO pretty confusing, because the client doesn't censor anything, it just records what to censor. I can not immediately come up with a great name, but maybe something like `secretrecordingclient` might be better?

Also, one sentence of godoc explaining this would be great